src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java

```diff
@@ -27,12 +27,12 @@
     public static final String COMMAND_WORD = "edit appointment";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits an appointment in the app. \n"
-            + "Parameters: INDEX"
+            + "Parameters: INDEX "
             + PREFIX_NAME + "NAME "
             + PREFIX_REMARK + "REMARK "
             + PREFIX_DATE + "DATE "
             + PREFIX_TIME + "TIME \n"
```

Comment:
Do you need to add the square brackets here too?

Reply:
oh yeah! nice! thanks! added it!